SnoopCompileBot/Project.toml

```diff
@@ -0,0 +1,31 @@
+name = "SnoopCompileBot"
+uuid = "1d5e0e55-7d74-4714-b8d8-efa80e938cf7"
+author = ["Tim Holy <tim.holy@gmail.com>"]
+version = "1.5.0"
+
+[deps]
+FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
+SnoopCompileAnalysis = "9ea4277c-da97-4c3a-afb0-537c066769de"
+SnoopCompileCore = "e2b509da-e806-4183-be48-004708413034"
+
+[compat]
+FilePathsBase = "0.9"
+YAML = "0.4"
+julia = "1"
+
+[extras]
```

Comment:
Does the bot need all these for its tests?

Reply:
Fixed in https://github.com/timholy/SnoopCompile.jl/pull/98/commits/0ebdb9648b2cd6d941178d71d1b0ecef6c94526b